packages/engines/src/index.ts

```diff
@@ -59,28 +58,13 @@ export { enginesVersion } from '@prisma/engines-version'
  */
 
 path.join(__dirname, '../query-engine-darwin')
```

Comment:
OT, but does it make sense to list the non Linux ones here? Isn't this just for deployment platforms that make sense, so mostly Linux?

Reply:
You may be right. According to `git blame`, these `path.join`s were added to address https://github.com/vercel/nft/issues/104, and Vercel has Linux only machines afaik

Reply:
This might be a bigger discussion we should bring to Pierre and others in a new issue, or Slack thread.

Reply:
@janpio @millsp I've created this issue: https://github.com/prisma/prisma-private/issues/207